app/src/main/java/org/astraea/partitioner/smoothPartitioner/SmoothWeightPartitioner.java

```diff
@@ -29,16 +31,26 @@
 
   private NodeLoadClient nodeLoadClient;
 
+  private long lastTime = -1;
+  private final Random rand = new Random();
+  private final LoadPoisson loadPoisson = new LoadPoisson();
+
+  @Override
+  public void onNewBatch(String topic, Cluster cluster, int prevPartition) {
+    Partitioner.super.onNewBatch(topic, cluster, prevPartition);
```

Comment:
why? it has empty body

Reply:
I don't know why he ran out.I don't need to instance it.so i will remove it.